pcdsdevices/interface.py

```diff
@@ -34,15 +34,12 @@
 logger = logging.getLogger(__name__)
 engineering_mode = True
 
-OphydObject_whitelist = ["name", "connected", "check_value", "log"]
-BlueskyInterface_whitelist = ["trigger", "read", "describe", "stage",
-                              "unstage"]
-Device_whitelist = ["read_attrs", "configuration_attrs", "summary",
-                    "wait_for_connection", "stop", "get", "configure"]
-Signal_whitelist = ["value", "put"]
+OphydObject_whitelist = []
+BlueskyInterface_whitelist = []
+Device_whitelist = ["stop"]
+Signal_whitelist = ["value", "put", "get"]
 Positioner_whitelist = ["settle_time", "timeout", "egu", "limits", "move",
-                        "position", "moving", "set_position",
-                        "set_current_position"]
+                        "position", "moving", "set_current_position"]
```

Comment:
I understand that there are requests to cut down on tabs here, but the following removals need a few words of justification:
- `name` is a pretty critical attribute for understanding the bluesky scans
- `connected` is immensely useful for understanding if your object's IOC is running properly
- `set_position` is an alias we were asked to add- only to be asked to remove it??

Reply:
The removal of attrs and all those that Zach mentioned surprised me a bit - scientists don't tweak attrs for bluesky scanning?

We may have been including too much to start with, but overall this strikes me as now including too little. Where the balance is in the middle, I don't really know, to be fair.

Reply:
@dlzhu Can you comment on all the tab whitelist changes?